app/build.gradle

```diff
@@ -27,6 +27,8 @@ android {
         ndk {
             moduleName "xtrs"
         }
+
+        vectorDrawables.generatedDensities = ['mdpi','hdpi','xhdpi']
```

Comment:
I have begun to migrate images to vector assets in the past but so far didn't use this gradle directive. Is it necessary?